pages/careers/index.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import { Layout } from '../../layouts/article/layout';
+
+const Header = () => (
+  <section className="header-main">
+    <article className="content">
+      <h1>Työpaikat</h1>
+      <p>Etsimme aina uusia kyvykkäitä ja innostuneita jäseniä tiimiimme.</p>
+      <p>
+        Haluaisimme kasvattaa pientä yhdeksän hengen porukkaamme. Olemme viime
+        vuoden loppupuolella perustettu konsulttilafka, jonka ytimessä on kova
+        osaaminen ja työntekijälähtöinen meisinki. Uskomme ketterään
+        ohjelmistokehitykseen, yhdessä tekemiseen, positiiviseen asenteeseen ja
+        jatkuvaan kehittymiseen.
+      </p>
+      <p>
+        Yritämme tarjota työntekijöillemme parhaan mahdollisen ympäristön
+        työskentelyyn ja oppimiseen, ja uskomme, että asiakasprojektit täytyy
+        miettiä tarkasti kullekin työntekijälle: projektin täytyy olla mieluisa
+        ja tukea työntekijän uraa ja kehitystä.
+      </p>
+      <p>
+        Alla näet avoimet työpaikkamme. Meillä on myös aina avoin rekry auki
+        osaaville tekijöille, joten otathan yhteyttä{' '}
+        <strong>rekry@rare.fi</strong> jos työskenteleminen Rarella kiinnostaa!
+      </p>
+    </article>
+
+    <style jsx>{`
+      .header-main {
+        display: flex;
+      }
+      .content {
+        width: 60%;
+      }
+      .image {
+        display: flex;
+        align-items: center;
+        width: 40%;
+      }
+      @keyframes animate {
```

Comment:
aika non-issue mut näis ei oo vissii mitää animoitavaa näillä sivuilla

Reply:
joo mäpäs siivoon pois nää kaikki käyttämättömät tyylit

Reply:
very good catch, olin vaa cobybastennu ne tuolt trainings sivuilt :D